### PR TITLE
Task 209168 feat: configurable appliance

### PR DESCRIPTION
**Pull request by deepali-chavhan**

### CODE REVIEW

File: src/app/demo/components/config-app/global-config/global-config.component.ts [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+})
+export class GlobalConfigComponent implements OnInit {
+    authoringConfig: any;
+    selectedOption: any;
+    configIdList = [];
+    messages!: Message[];
+    value: boolean = false;
+    addAuthoringForm: FormGroup;
+
+    languageList = [
```
Comment (gouravmore) [inline src/app/demo/components/config-app/global-config/global-config.component.ts]: can we create config interface for this

File: package.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
+{
+    "name": "content-service-ui",
+    "version": "16.0.0",
+    "license": "PrimeNG Commercial",
```
Comment (gouravmore) [inline package.json]: This we need to check - please remove this for now

File: src/app/app-routing.module.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        RouterModule.forRoot([
+            {
+                path: '',
+                pathMatch: 'full',
+                redirectTo: 'content/global-config/list'
+              }, 
+            {
+                path: 'content', component: AppLayoutComponent,
+                children: [
+                    { path: 'global-config/list', loadChildren: () => import('./demo/components/config-app/global-config/global-config.module').then(m => m.GlobalConfigModule) },
```
Comment (gouravmore) [inline src/app/app-routing.module.ts]: folder name should not be demo

File: src/app/app.component.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+@Component({
+    selector: 'app-root',
+    templateUrl: './app.component.html'
+})
+export class AppComponent implements OnInit {
+
+    constructor(private primengConfig: PrimeNGConfig) { }
+
+    ngOnInit() {
+        this.primengConfig.ripple = true;
```
Comment (gouravmore) [inline src/app/app.component.ts]: is this required?

File: src/app/demo/components/config-app/story-list/add-or-edit-story/add-or-edit-story.component.html [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+
+      <!-- <div class="field">
+                <label for="language">{{ 'Image' }} *</label>
+                <input type="text" name="image" pInputText id="image" autofocus placeholder="{{ 'Enter image' }}" formControlName="image" (input)="onInputChange($event,'org')" />
+                <small *ngIf="submitted && addStoryForm.controls['image'].errors">{{ 'image' }}</small>
+            </div> -->
+    </div>
+    <div class="p-col-12 p-md-6 p-lg-4 p-xl-3 p-button-container">
+      <button
+        (click)="saveOrg('Add')"
```
Comment (gouravmore) [inline src/app/demo/components/config-app/story-list/add-or-edit-story/add-or-edit-story.component.html]: what is saveOrg?

File: src/app/demo/components/config-app/story-list/add-or-edit-story/add-or-edit-story.component.html [hunk 6]
```diff
@@ -0,0 +1,194 @@
...
+
+      <!-- <div class="field">
+                <label for="image">{{ 'Image' }} *</label>
+                <input type="text" name="image" pInputText id="image" autofocus placeholder="{{ 'Enter image' }}" formControlName="image" (input)="onInputChange($event,'org')" />
+                <small *ngIf="submitted && editStoryForm.controls['image'].errors">{{ 'image' }}</small>
+            </div> -->
+    </div>
+    <div class="p-col-12 p-md-6 p-lg-4 p-xl-3 p-button-container">
+      <button
+        (click)="saveOrg('Edit')"
```
Comment (gouravmore) [inline src/app/demo/components/config-app/story-list/add-or-edit-story/add-or-edit-story.component.html]: need to update the method name

File: src/app/demo/components/config-app/story-list/add-or-edit-story/add-or-edit-story.component.ts [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+    editStoryForm: FormGroup;
+    messages: Message[];
+    CurrentChannelName: string;
+    difficulty: string;
+    difficultyLvlList = [
+        { label: 'LOW', value: 'low' },
+        { label: 'MEDIUM', value: 'medium' },
+        { label: 'DIFFICULT', value: 'difficult' },
+    ];
+    languageList = [
```
Comment (gouravmore) [inline src/app/demo/components/config-app/story-list/add-or-edit-story/add-or-edit-story.component.ts]: use interface for config

File: src/styles.scss [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+
+$gutter: 1rem; //for primeflex grid system
+@import "assets/layout/styles/layout/layout.scss";
+
+/* PrimeNG */
+@import "../node_modules/primeng/resources/primeng.min.css";
+@import "../node_modules/primeflex/primeflex.scss";
+@import "../node_modules/primeicons/primeicons.css";
+
+/* Demos */
```
Comment (gouravmore) [inline src/styles.scss]: this comments can be removed

File: src/app/demo/components/config-app/global-config/global-config.component.ts (hunk 1 above)
Comment (deepali-chavhan) [reply src/app/demo/components/config-app/global-config/global-config.component.ts]: config interface created.

File: package.json (hunk 2 above)
Comment (deepali-chavhan) [reply package.json]: removed for now.

File: src/app/app-routing.module.ts (hunk 3 above)
Comment (deepali-chavhan) [reply src/app/app-routing.module.ts]: name changed to configurable-appliance

File: src/app/app.component.ts (hunk 4 above)
Comment (deepali-chavhan) [reply src/app/app.component.ts]: Committed by mistakenly. removed in the next PR.

File: src/app/demo/components/config-app/story-list/add-or-edit-story/add-or-edit-story.component.html (hunk 5 above)
Comment (deepali-chavhan) [reply src/app/demo/components/config-app/story-list/add-or-edit-story/add-or-edit-story.component.html]: Name change with addOrEditStory.

File: src/app/demo/components/config-app/story-list/add-or-edit-story/add-or-edit-story.component.ts (hunk 7 above)
Comment (deepali-chavhan) [reply src/app/demo/components/config-app/story-list/add-or-edit-story/add-or-edit-story.component.ts]: Created new config for all.

File: src/styles.scss (hunk 8 above)
Comment (deepali-chavhan) [reply src/styles.scss]: removed.

**Pull request merged**